widget/app/public/index.html

```diff
@@ -7,6 +7,7 @@
       content="width=device-width, initial-scale=1, maximum-scale=1"
     />
     <title>Rango Widget</title>
+    <link rel="manifest" href="./manifest.json" />
```

Comment:
This manifest file is fine. The reason that still doesn't work with safe site is that parcel is transforming `manifest.json`to `manifest.webmanifest` during the build phase and safe web app doesn't accept it.

You should check parcel document to avoid it or you could update build script to re-transform it. (for test purpose)

something like this:
`    "build": "parcel build --cache-dir=.parcel-cache && sed -i 's/manifest.webmanifest/manifest.json/g' ./dist/index.html && cp ./dist/manifest.webmanifest ./dist/manifest.json",
`

Reply:
I tried that with placing manifest.json in dist directory and changing index.html file but it didn't work.